version: '3.7'

services:
  sage:
    image: sagemath/sagemath:9.1
    ports:
      - "8888:8888"
    volumes:
      - ./data:/home/sage
    environment:
      - PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
      - HOME=/home/sage
    command: ["sage-jupyter"]
 